cmd/wallet-web/package.json

```diff
@@ -16,8 +16,8 @@
     "test": "npm run test:setup && npm run test:start && npm run test:teardown"
   },
   "dependencies": {
-    "@trustbloc-cicd/agent-sdk-web": "0.1.8-snapshot-d91b3fa",
-    "@trustbloc-cicd/wallet-sdk": "0.1.8-snapshot-d91b3fa",
+    "@trustbloc-cicd/agent-sdk-web": "^0.1.8-snapshot-67d81b9",
+    "@trustbloc-cicd/wallet-sdk": "^0.1.8-snapshot-67d81b9",
```

Comment:
Should we just keep exact version since we specify commit anyways?

Reply:
done